src/App/Components/Popular/bottom-popular.js

```diff
@@ -11,31 +11,31 @@ const BottomPopular = (props) => {
   const { articles } = props;
```

Comment:
@MCFrank16 please, check all iterations of these urls

Reply:
I did it @nkpremices you can check new changes I've pushed